tools/install-kube-builder.sh

```diff
@@ -0,0 +1,10 @@
+os=$(go env GOOS)
+arch=$(go env GOARCH)
+
+# download kubebuilder and extract it to tmp
+curl -L https://go.kubebuilder.io/dl/2.3.1/${os}/${arch} | tar -xz -C /tmp/
+
+# move to a long-term location and put it on your path
+# (you'll need to set the KUBEBUILDER_ASSETS env var if you put it somewhere else)
+sudo mv /tmp/kubebuilder_2.3.1_${os}_${arch} /usr/local/kubebuilder
```

Comment:
This is going to overwrite any existing binary. If you want to ensure it's a specific version for the purpose of this repo, put it somewhere under the working directory and set environment entries (KUBEBUILDER_ASSETS and possibly PATH) in other scripts/Makefiles.

Reply:
Refactored to use the pre-built action provided by `fluxcd/pkg`